nitrogen/src/display.rs

```diff
@@ -64,6 +87,7 @@ impl Display {
         &mut self,
         device: &DeviceContext,
         res_list: &mut ResourceList,
+        swapchain_setup_config: &SwapchainSetupConfig,
```

Comment:
Now that the type is `Copy` I think we can pass it by value instead of using a reference.